docs/topics/gradle.md

```diff
@@ -966,6 +966,55 @@ Here is a complete list of options for Gradle tasks:
 | `target` | Generate JS files for specific ECMA version | "v5" | "v5" |
 | `typedArrays` | Translate primitive arrays to JS typed arrays |  | true |
 
+#### Example of additional arguments usage
+
+Use the attribute `freeCompilerArgs` to supply additional (including experimental) compiler arguments. You can add a single 
+argument to this attribute or a list of arguments divided by a comma:
```

Comment:
This option has type `List<String>`, so "a list of arguments divided by a comma" doesn't make sense

Reply:
Please. check now.